developers/feegrant-for-blobs.md

```diff
@@ -0,0 +1,142 @@
+# FeeGrant module for blobs submission
+
+## Overview
+
+This guide provides developers with the knowledge to use the FeeGrant
+module on the Celestia's Mocha testnet chain for granting a data
+availability node's account to submit blobs without constantly
+funding it, enabling a third-party account to cover the transaction fees.
+
+## Pre-requisites
+
+- celestia-app CLI installed
+- celestia-node and CLI tools installed
+- Access to a Mocha node (e.g., `https://rpc.celestia-mocha.com:443`)
+- Running DA Light node on Mocha testnet
+- Separate Celestia account with sufficient funds
+
+## Introduction
+
+Each DA node contains a Celestia account that is used to pay for blobs
+submissions. In order to unify the fee payment process, the FeeGrant module
+allows a third-party account to pay for the fees incurred by the DA node's
+account. This module is useful for projects that require a third-party to
+cover the fees for the DA node's account. You will need one account that will
+contain the funds and another account that will be in the DA node. You will
+see the DA node's account once you initialise the node.
+
+## Granting fee allowances
+
+To grant fee allowances, allowing a third-party (granter) account to pay
+for the fees incurred by a Celestia data availability node (grantee)
+account, use the following command:
+
+```bash
+celestia-appd tx feegrant grant \
+  [grantee_address] [granter_address] \
+  --home [path_to_grantee] \
```

Comment:
should this be `[path_to_granter]` @Bidon15 ?

Reply:
I cannot get it to work with:

```bash
celestia-appd tx feegrant grant \
  celestia18uguxd6avku8zrcugmlvjcfnm7wuc53ej2ujhn celestia1spgm03y9ctpnw9lzqpuxrf6qvwszrqtnq4sjs7 \
  --home ~/.celestia-light-mocha-4/ \
  --node https://rpc.celestia-mocha.com:443 \
  --spend-limit 1000000utia \
  --allowed-messages "/cosmos.bank.v1beta1.MsgSend,/celestia.blob.v1.MsgPayForBlobs" \
  --chain-id mocha-4 \
  --keyring-backend test \
  --fees 20000utia \
  --broadcast-mode block \
  --yes
Error: key with address celestia18uguxd6avku8zrcugmlvjcfnm7wuc53ej2ujhn not found: key not found
```

Where `celestia18uguxd6avku8zrcugmlvjcfnm7wuc53ej2ujhn` is grantee (light node account) and `celestia1spgm03y9ctpnw9lzqpuxrf6qvwszrqtnq4sjs7` is the celestia-app account. I've tried different paths:
1. `--home ~/.celestia-light-mocha-4/`
2. `--home ~/.celestia-light-mocha-4/data/keys/`
3. `--home ~/.celestia-light-mocha-4/data/keys/keyring-test/josh.info`
4. `--home ~/.celestia-light-mocha-4/data/keys/keyring-test/`